cli/templatepull.go

```diff
@@ -130,6 +131,17 @@ func (r *RootCmd) templatePull() *clibase.Cmd {
 				dest = templateName
 			}
 
+			clean, err := filepath.Abs(filepath.Clean(dest))
+			if err != nil {
+				cliui.Error(inv.Stderr, fmt.Sprintf("cleaning destination path %s failed: %q", dest, err))
+				return err
+			}
+
+			if dest != clean {
+				cliui.Warn(inv.Stderr, fmt.Sprintf("cleaning destination path from %s to %s", dest, clean))
```

Comment:
nit: I wouldn't warn users here, it the tool can fix the path then we don't need to them

Reply:
I was thinking this process of cleaning the dir might cause unexpected results, but on reflection I think you're right; I've removed it.